modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -13,5 +13,12 @@ To collect {ProductName} data with `must-gather`, you must specify the {ProductN
 
 [source,terminal]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7
+$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8
+----
+
+To collect {ProductName} data for a specific control plane namespace with `must-gather`, you must specify the {ProductName} image and namespace. In this example, replace  `<namespace>` with your control plane namespace, such as `istio-system`.
```

Comment:
In the previous code snippet, you introduced it with a colon, `...Service Mesh image:` 

So wouldn't you need to introduce this code snippet with a colon too? So, it'd be, `...such as `istio-system`:`